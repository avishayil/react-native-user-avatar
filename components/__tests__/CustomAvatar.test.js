/* eslint max-len: 0 */

import 'react-native';
import {Image} from 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import {CustomAvatar} from '../';

it('should render <CustomAvatar /> with local image reference', () => {
  const tree = renderer.create(<CustomAvatar component={<Image source={require('./images/rn_logo.png')} style={{ width: 50, height: 50}} />} size={50} />).toJSON();
  expect(tree).toMatchSnapshot();
});
