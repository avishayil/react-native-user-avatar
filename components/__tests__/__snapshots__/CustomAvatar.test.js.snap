// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <CustomAvatar /> with local image reference 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "column",
      "height": 50,
      "justifyContent": "center",
      "marginTop": -2.5,
      "width": 50,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../components/__tests__/images/rn_logo.png",
      }
    }
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
  />
</View>
`;
